osf/models/node.py

```diff
@@ -1156,13 +1156,15 @@ def visible_contributors(self):
             contributor__visible=True
         ).order_by('contributor___order')
 
-    def get_spam_fields(self, saved_fields):
+    def get_spam_fields(self, saved_fields=None):
```

Comment:
Add a comment here to explain the abuse (we can force spam check any fields we want by setting "saved_fields"), maybe with a TODO to change the way we spam check Wikis.